hardhat.config.js

```diff
@@ -55,10 +55,15 @@ const argv = require('yargs/yargs')()
     },
   }).argv;
 
-require('@nomicfoundation/hardhat-toolbox');
+// "implies" doesn't work correctly. we force gas if gasReport is set
+argv.gas ||= !!argv.gasReport;
```

Comment:
Pretty weird it's not working as expected. I'll take a look to understand what's going on here. We shouldn't merge it if we're misinterpreting how it works.

Otherwise, we may remove the `implies`.